File: src/main/java/com/thebluehats/server/api/daos/PerformanceStatsService.java
package com.thebluehats.server.api.daos;

import org.bukkit.entity.Player;

public interface PerformanceStatsService {
    int getPlayerXp(Player player);

    void setPlayerXp(Player player, int value);

    double getPlayerGold(Player player);

    void setPlayerGold(Player player, double value);
}

File: src/main/kotlin/com/thebluehats/server/api/daos/PerformanceStatsService.kt
package com.thebluehats.server.api.daos

import org.bukkit.entity.Player

interface PerformanceStatsService {
    fun getPlayerXp(player: Player): Int
    fun setPlayerXp(player: Player, value: Int)
    fun getPlayerGold(player: Player): Double
    fun setPlayerGold(player: Player, value: Double)
}